sources/packages/backend/libs/integrations/src/esdc-integration/sin-validation/sin-validation-processing.service.ts

```diff
@@ -29,20 +31,18 @@ export class SINValidationProcessingService {
     private readonly studentService: StudentService,
     private readonly sequenceService: SequenceControlService,
     private readonly sinValidationIntegrationService: SINValidationIntegrationService,
+    private readonly systemUsersService: SystemUsersService,
   ) {
     this.esdcConfig = config.esdcIntegration;
   }
 
   /**
    * Identifies all the students that still do not have their SIN
    * validated and create the validation request for ESDC processing.
-   * @param auditUserId user that should be considered the one that is
-   * causing the changes.
    * @returns result of the upload operation.
    */
-  async uploadSINValidationRequests(
-    auditUserId: number,
-  ): Promise<SINValidationUploadResult> {
+  async uploadSINValidationRequests(): Promise<SINValidationUploadResult> {
+    const auditUser = await this.systemUsersService.systemUser();
     this.logger.log("Retrieving students with pending SIN validation...");
     const students =
       await this.studentService.getStudentsPendingSinValidation();
```

Comment:
Not a part of the PR. But considering the volume of records from which this select is going to be returned, index seems to be essential here. @andrewsignori-aot @guru-aot @ann-aot @andrepestana-aot ?

We can create one ticket to analyze the selects which may need index and create.

Reply:
if getStudentsPendingSinValidation is the query are we talking about? if yes, then i feel indexing may not be the best option as we are not using a lot of conditions in the where, rather than checking for null. We can definitely revisit query to better understand the performance aspect to it.